src/schedule/zephyr_dma_domain.c

```diff
@@ -597,6 +619,15 @@ static int zephyr_dma_domain_unregister(struct ll_schedule_domain *domain,
 	return 0;
 }
 
+static bool zephyr_dma_domain_is_pending(struct ll_schedule_domain *domain,
+			  struct task *task, struct comp_dev **comp)
+{
+	if (task->state == SOF_TASK_STATE_PENDING)
+		return true;
+	else
+		return false;
```

Comment:
`return task->state == SOF_TASK_STATE_PENDING;`

Reply:
Thanks for the review. Will change this.